modules/unzip/main.nf

```diff
@@ -23,18 +23,19 @@ process UNZIP {
     path archive
 
     output:
-    path "*"             , emit: result
-    path "*.version.txt" , emit: version
+    path "unzipped_archive/" , emit: unzipped_archive
```

Comment:
Sorry for being such a PITA here, but I think with this version we could get into trouble if any process consumes two extracted archives. 

* Would `unzipped_archive/*` emit the directory, or just the files? 
* Or maybe something like "${archive.baseName}" as directory name?

Reply:
Perfect! I actually only thought of the second case myself.